src/tools/gophish_import.py

```diff
@@ -164,21 +166,25 @@ def build_campaigns(api, assessment):
             except Error as e:
                 if e.message == "Template name already in use":
                     logging.warning(
-                        f"{e}. Finding previously loaded template to delete."
+                        "%s. Finding previously loaded template to delete.", e.message
                     )
                     templates = api.templates.get()
                     logging.debug(
-                        f"Checking {len(templates)} for previously imported template to get ID"
+                        "Checking %i for previously imported template to get ID",
+                        len(templates),
                     )
                     for old_template in templates:
                         if old_template.name == new_template.name:
                             logging.debug(
-                                f"Deleting Template with ID {old_template.id}"
+                                "Deleting Template with ID %i", old_template.id
```

Comment:
Same question as above about `%d` or `%s`:
```suggestion
                                "Deleting Template with ID %d", old_template.id
```

Reply:
See aa93585